Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/FormattedError.kt

```diff
@@ -0,0 +1,13 @@
+package de.rki.coronawarnapp.util
+
+import android.content.Context
+
+interface FormattedError {
+    fun getFormattedError(context: Context): String
+}
+
+fun Throwable.tryFormattedError(context: Context): String = when {
```

Comment:
redundant return type specification

Reply:
I think specifying a return type is okay if it makes it easier to read (and prevents accidental type changes when using `when`). Unless you feel strongly about this, I'll leave it as is.